static/app/components/charts/baseChart.tsx

```diff
@@ -338,6 +340,7 @@ function BaseChartUnwrapped({
   transformSinglePointToBar = false,
   transformSinglePointToLine = false,
   onChartReady = () => {},
+  'data-test-id': dataTestId,
```

Comment:
Why does this need to be a prop? I dont think we should ever expose a testing utility as part of the component's API. Can we hardcode it or maybe infer it's value from the props?

Reply:
sometimes we pass `...props` down to other components and the destructure aims to avoid it. We are doing this in a couple of places in our app and maybe this is something we can discuss in the frontend tsc 😉

Reply:
Oh, ok. Smells like we are testing some details here